tools/xcodeproj-shims/xcodeproj-installer.sh

```diff
@@ -0,0 +1,26 @@
+# Copies the xcodeproject from the bazel output directory to the BAZEL_WORKSPACE directory when ran
+set -eu
+readonly project_path="${PWD}/$(project_short_path)"
+readonly dest="${BUILD_WORKSPACE_DIRECTORY}/$(project_short_path)/"
+readonly tmp_dest=$(mktemp -d)/$(project_full_path)/
+
+readonly stubs_dir="${dest}/bazelstubs"
+mkdir -p ${stubs_dir}
```

Comment:
```suggestion
mkdir -p "${stubs_dir}"
```
to prevent spaces from splitting args